packages/core/src/Button/Button.tsx

```diff
@@ -143,6 +153,24 @@ const variations = {
       box-shadow: ${(props) => ` 0 0 0 2px  ${getPaletteColor(props.disabled ? '' : 'dark')(props)}`};
     }
   `,
+  input: css`
+    appearance: none;
+    background-color: transparent;
+    border-style: solid;
+    border-width: 1px;
+    color: ${getPaletteColor('text.base')};
+    display: block;
+    font-family: inherit;
+    font-weight: normal;
+    line-height: normal;
+    margin: 0;
+    padding: 14px 12px;
```

Comment:
Should we allow for this padding to be configurable?

Reply:
`${space}` a few lines below means it still allows it, this is just a default value for it 😄

Reply:
Missed that 👍 thanks